### PR TITLE
Backticks for code blocks

### PR DESCRIPTION
**Pull request by atesgoral**
Converts stuff between `` ` `` and `` ` `` to `<code>` blocks, except when a `` ` `` is escaped with a `` \` ``

### CODE REVIEW

**Comment by atesgoral** [comment]
Closes #247

File: dwitter/static/main.css [hunk 1]
```diff
@@ -196,6 +196,10 @@ ul.messages li.success{
   word-wrap: break-word;
 }
 
+.comment-message code {
+  background: #eee;
```
Comment (lionleaf) [inline dwitter/static/main.css]: Might want to change font too?

Comment (atesgoral) [reply dwitter/static/main.css]: The font is already set by the top-level `code` CSS rule. Same font as the editor. Might want to play around with adding a border + padding around, but then layout might get thrown off.

Comment (lionleaf) [reply dwitter/static/main.css]: Oh, right. Looks pretty good already :)

**Comment by lionleaf** [comment]
live!

**Pull request merged**